dynflow.gemspec

```diff
@@ -29,7 +29,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency "rack-test"
   s.add_development_dependency "minitest"
   s.add_development_dependency "minitest-reporters"
-  s.add_development_dependency "activerecord"
+  s.add_development_dependency "activerecord", '< 5.0.0'
```

Comment:
http://guides.rubygems.org/specification-reference/ that doesn't seem to exist, unfortunately